source/extensions/filters/http/oauth2/filter.h

```diff
@@ -148,11 +146,17 @@ class FilterConfig {
     return default_refresh_token_expires_in_;
   }
   bool disableIdTokenSetCookie() const { return disable_id_token_set_cookie_; }
+  const OptRef<const RouteRetryPolicy> retryPolicy() const {
+    if (!retry_policy_.has_value()) {
+      return absl::nullopt;
+    }
+    return makeOptRef(retry_policy_.value());
+  }
```

Comment:
Not sure if this is the correct way to convert an `absl::optional<RouteRetryPolicy>` to `OptRef<const RouteRetryPolicy>` @wbpcode :-)